slices/web/lib/web/action.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
+# auto_register: false
 
-require_relative "./actions/authentication"
+require "hanami/action"
```

Comment:
Again, "require what you require" when it comes to making sure your superclass is around.